docs/concepts/usage-ideas-examples.md

```diff
@@ -72,7 +72,7 @@ By storing small scripts or big databases of your project on IPFS, depending on
 
 On some heavy analysis, you can benefit from parallelizing calculations on several nodes using Hadoop, for example. But on very large datasets, fetching the relevant subset for each node to compute is longer than the actual computation. Scott Brisbane proposed a design in a [thesis](https://s3-ap-southeast-2.amazonaws.com/scott-brisbane-thesis/decentralising-big-data-processing.pdf) to drastically reduce the fetching time using IPFS and divided the overall analysis time by four. 
 
-### Deadman switch
```

Comment:
Nit: what a weird / interesting term... anyway, I'm seeing that this is usually written as "Dead man's switch" or "Deadmans switch" or "Deadman switch" but not "Dead-man switch".

Reply:
Not actually sure what the best one is, but probably best to just pick one. "Dead man's switch" showed up at the top of search results

Reply:
I'm gonna go with "Dead man's switch", that seems to be the most common spelling.